app/adapters/alma_adapter/bib.rb

```diff
@@ -38,12 +38,20 @@ def get_bib_records(ids, conn=nil, opts={})
       # Returns list of holding records for a given MMS
       # @params id [string]. e.g id = "991227850000541"
       def get_holding_records(id)
-        res = Alma::Connector.connection.get "bibs/#{id}/holdings", {
+        res = AlmaAdapter::Connector.connection.get "bibs/#{id}/holdings", {
           :apikey => self.apikey
         }
         doc = res.body
       end
 
+      # @params id [string]. e.g id = "991227850000541"
+      # @return [Hash] of holdings / items data
+      def get_items_for_bib(id)
+        opts = { limit: 100, expand: "due_date_policy,due_date" }
+        Alma::BibItem.find(id, opts)
+        # todo: transform alma response to marc liberation format
```

Comment:
I understand this is a to do, but do you need to add the `:api_key` we configure for bibs_read_only ?

Reply:
We have different api_keys based on the request we want to do.

Reply:
Please ignore this for now. I asked @mzelesky if we can create one api_key in the sandbox. Otherwise we will update the config so that we include multiple api_keys.

Reply:
The gem passes the api key in the header as opposed to in the url. I made the stub more explicit to show what it looks like. I also reworked the config setup a little bit. We can refactor the AlmaAdapter config in the same way (we could just get rid of that initializer and use `Rails.configuration.alma` instead) but I didn't want to introduce conflicts if you were working on any config stuff.

If we have multiple keys we may need to just use the one with the highest required level of permissions.

I'll test the rest of the integration out when I get the parsing done and can deploy it to staging.

Reply:
Yes I agree; With the current keys we couldn't pick one with the highest permissions because it doesn't exist. We can discuss this tomorrow at the checkin with the rest of the group. I think for now we can use the one we have configured and is only for the bibliographic call and read only.

Reply:
@hackmastera the alma_keys need to be added on bibdata-staging in order to test it.